activerecord/lib/active_record/core.rb

```diff
@@ -639,14 +639,14 @@ def strict_loading?
     #   user.comments
     #   => ActiveRecord::StrictLoadingViolationError
     #
-    # === Parameters:
+    # ==== Parameters
     #
     # * value - Boolean specifying whether to enable or disable strict loading.
     # * mode - Symbol specifying strict loading mode. Defaults to :all. Using
     #          :n_plus_one_only mode will only raise an error if an association
     #          that will lead to an n plus one query is lazily loaded.
     #
-    # === Example:
+    # ==== Example
```

Comment:
Was the additional heading intentional?

Reply:
Yes, we always use `==== Example` instead of `=== Example`.

Reply:
Oh, I asked because I saw this one in the diff but maybe you just missed it?

https://github.com/rails/rails/pull/47312/files#diff-2cedfc1813acfb5212a27d06277b604a9e213088dce9749526d3de24ae60a585

I haven't grep'd for other variations. 👀